src/debug/HyprCtl.cpp

```diff
@@ -383,7 +383,7 @@ R"#(    {
         "name": "%s"
     },)#",
                 &d,
-                d.pWlrDevice ? d.pWlrDevice->name : ""
+                d.name
```

Comment:
std::string cannot be used directly in C format functions. You need to call `.c_str()` (also applies to all of the below)

Reply:
Ah, thanks! Didn't notice this.